install/saaj/docs/SAAJ2.0-ReleaseNotes.html

```diff
@@ -77,24 +77,23 @@ <h2><a name="platform">Platform Notes</a></h2>
 <p>The Jakarta SOAP Attachments TCK tests run on the following platforms:</p>
 
 <ul>
-<li>Oracle Linux 7.1</li>
-<li>Windows 10</li>
+<li>CentOS 7</li>
```

Comment:
Linux would be more generic than CentOS, could we use that?

Reply:
Other changes look fine to me.

Reply:
This was discussed on https://github.com/eclipse-ee4j/jakartaee-tck/pull/380 too. Think CentOS 7 is OK.